### PR TITLE
Feature/linting setup

### PR DESCRIPTION
**Pull request by julia-rom**
- added basic configs for typescript, prettier, and eslint
- create react app cleanup

### CODE REVIEW

File: mrtt-ui/.eslintrc.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  ],
+  parser: '@typescript-eslint/parser',
+  parserOptions: {
+    ecmaFeatures: {
+      jsx: true
+    },
+    ecmaVersion: 'latest',
+    sourceType: 'module'
+  },
+  plugins: ['react', '@typescript-eslint'],
```
Comment (JBurkinshaw) [inline mrtt-ui/.eslintrc.js:22]: I think 'react-hooks', 'testing-library',  and 'jest-dom' will be useful here. Can you add `react-hooks` for now and we can add the rest when the time comes?

File: mrtt-ui/.eslintrc.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    browser: true,
+    es2021: true,
+    node: true,
+    jest: true
+  },
+  extends: [
+    'eslint:recommended',
+    'plugin:react/recommended',
+    'plugin:@typescript-eslint/recommended',
+    'plugin:prettier/recommended'
```
Comment (JBurkinshaw) [inline mrtt-ui/.eslintrc.js:12]: We can add to this as needed later. `react-app/jest` should probably added when we add testing libraries.

**Review by JBurkinshaw — CHANGES_REQUESTED** [review]

File: mrtt-ui/.eslintrc.js (hunk 1 above)
Comment (julia-rom) [reply mrtt-ui/.eslintrc.js:22]: yup, added!

File: mrtt-ui/.eslintrc.js (hunk 2 above)
Comment (julia-rom) [reply mrtt-ui/.eslintrc.js:12]: will add!

**Review by JBurkinshaw — APPROVED** [review]

**Pull request merged**